aws_protocol_adaptor/device_client/aws_nvmsgbroker.c

```diff
@@ -39,11 +39,11 @@ NvDsMsgApiErrorType (*nvds_msgapi_send_ptr)(NvDsMsgApiHandle conn, char *topic,
 NvDsMsgApiErrorType (*nvds_msgapi_disconnect_ptr)(NvDsMsgApiHandle h_ptr);
 static GMutex thread_mutex;
 static GQueue *work_queue;
-static struct timespec last_send_time_stamp;	   // this is to make sure we send or yield frequent enough so we do not get disconnected.
-static nvds_msgapi_connect_cb_t disconnect_cb;	   // disconnect handler provided by connect thread
+static struct timespec last_send_time_stamp; // this is to make sure we send or yield frequent enough so we do not get disconnected.
+static nvds_msgapi_connect_cb_t disconnect_cb; // disconnect handler provided by connect thread
 static nvds_msgapi_subscribe_request_cb_t nvds_cb; // msgapi subscribe callback handler
 static char *subscribed_topics[MAX_SUBSCRIPTIONS]; // to store the subscribed topics in order to be used during unsubscribe operation
-static size_t num_subscriptions = 0;			   // initializing the number of topics subscribed variable
+static size_t num_subscriptions = 0; // initializing the number of topics subscribed variable
```

Comment:
You don't need to tell us that you're initializing a variable—that's patently obvious. What we want to know is what the variable means.

```suggestion
static size_t num_subscriptions = 0; // number of registered subscriptions
```